.cirrus.yml

```diff
@@ -124,6 +124,27 @@ task:
     - name: web_tests-7_last-linux # last Web shard must end with _last
       << : *WEB_SHARD_TEMPLATE
 
+    - name: web_integration_test_linux
+      compile_host_script: |
+        cd $ENGINE_PATH/src
+        ./flutter/tools/gn --unoptimized --full-dart-sdk
+        ninja -C out/host_debug_unopt
+      fetch_framework_script: |
+        cd $ENGINE_PATH/src/flutter/tools
+        ./clone_flutter.sh
+        cd $FRAMEWORK_PATH/flutter
+        bin/flutter update-packages --local-engine=host_debug_unopt
+      script:
+        - git clone https://github.com/flutter/web_installers.git
+        - cd web_installers/packages/web_drivers/
+        - $ENGINE_PATH/src/third_party/dart/tools/sdks/dart-sdk/bin/pub get
+        - $ENGINE_PATH/src/third_party/dart/tools/sdks/dart-sdk/bin/dart lib/web_driver_installer.dart chromedriver --install-only
+        - ./chromedriver/chromedriver --port=4444 &
+        - cd $ENGINE_PATH/src/flutter/e2etests/web/text_editing
+        - $FRAMEWORK_PATH/flutter/bin/flutter config --local-engine=host_debug_unopt --no-analytics --enable-web
+        - $FRAMEWORK_PATH/flutter/bin/flutter pub get --local-engine=host_debug_unopt
+        - $FRAMEWORK_PATH/flutter/bin/flutter drive -v --target=test_driver/text_editing_e2e.dart -d web-server --release --browser-name=chrome --local-engine=host_debug_unopt
```

Comment:
We're expecting to add more test files in the future, right? It might make sense to split this out into a script, and maybe even shard it if the tests take awhile to run. I guess this could be added incrementally in the future.

Reply:
Yes I agree. This is configuring web as well. I'll make a script with multiple arguments (so we can add mobile there too)

Reply:
Would it make sense to add running e2e tests into the `felt` tool? We do that already for unit-tests and goldens.

Reply:
Yes I'm planning to that. either felt or a new tool. I'll do it depending on the results of the sheet I sent you. (See my answer to mdebbar's comment)